features/search/default-queries-config-file-clientid.feature

```diff
@@ -0,0 +1,59 @@
+@sapi3
+Feature: Test the Search API v3 default queries from the config file when using an clientId
+
+  # test_client_with_default_search_query has a default filter with 'regions:nis-24134' in the config file
```

Comment:
Only one small remark, maybe we need to change the default query to make sure it is the client ID default query and not the API key default query?

Reply:
Good remark will do it like that.